invenco-initiated-messages/proof-of-play/proof-of-play.md

```diff
@@ -29,6 +29,18 @@
     - partial
 - **siteId** - _required_ - The Invenco name of the site.
 - **terminalId** - _required_ - The id of the terminal at the site.
+- **couponStatus** - _optional_ -  A string that will indicate if the coupon printing was initiated. **Required** if coupon information is available.
+  - Values
+    - 0 - OK
+    - 2 - printer offline
+    - 3 - printer unknown error
+    - 4 - printer over temp
+    - 5 - printer under temp
+    - 6 - printer paper out
+    - 7 - printer paper jammed
+    - 8 - printer cover opened
+    - _N_ - _reason goes here_
```

Comment:
how long can this be?  We have to be careful of making the pop messages too large.

Reply:
future messages are unlikely to be much longer than these - ~25 characters or so. If we need to put a limit in I'd suggest maybe 40 to be safe. In practice, I don't think these will significantly have an effect on total POP traffic since the field is only present if a customer actually pressed the key to print the coupon - I'd expect this would be a pretty low percentage of impressions.

Reply:
I know that the redemption rates tend to be a low single-digit percentage so I think it would be a tiny fraction of all PoP data that have coupon properties.

Reply:
Okie